src/questions.json

```diff
@@ -102,7 +102,7 @@
     },
     {
       "prompt": "ף",
-      "answers": ["fay sofit", "final fay", "fei sofit", "final fei"]
+      "answers": ["fay sofit", "final fay", "fei sofit", "final fei", "pay sofit", "final pay", "pei sofit", "final pei"]
```

Comment:
can we add a pay sofit as a letter instead?

Reply:
I don't follow 😅 ... like a `Fay` but with a `Dagesh`? it is not really used in Hebrew as far as I know